src/test/osdc/object_cacher_stress.cc

```diff
@@ -47,7 +47,7 @@ class C_Count : public Context {
     : m_op(op), m_outstanding(outstanding) {}
   void finish(int r) override {
     m_op->done++;
-    assert(m_outstanding > 0);
+    assert(*m_outstanding > 0);
     *m_outstanding--;
```

Comment:
@joscollin Yeah, not sure why (I must have touched it, sometime! :>), but either way, happy to be involved. I agree with your analysis, that's what I found as well.